src/map/Map.js

```diff
@@ -1299,6 +1299,8 @@ export var Map = Evented.extend({
 		this._fireDOMEvent(e, type);
 	},
 
+	_nonBubblingEvents: ['click', 'dblclick', 'mouseover', 'mouseout', 'contextmenu'],

```

Comment:
Could we just call this `_mouseEvents` or similar? The mix of the option `bubblingEvents` and `nonBubblingEvents` in the condition below made my head spin for a while.